extensions/divina.md

```diff
@@ -172,7 +179,158 @@ This current draft does not cover guided navigation over alternate versions of e
 ]
 ```
 
-## 5. Package
+## 5. Transitions
```

Comment:
I would be much more comfortable if this was moved to a new document under `extensions/transitions`.

Reply:
we will do that among other redistribution of information in the RWPM documentation.

Reply:
I don't see the harm in doing it right now for transitions, quite the opposite in fact.

We're not going to do a massive PR for the other documents, it's going to be handled one by one.

Reply:
Let's proceed with the re-distribution of information next week.